src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -821,6 +821,22 @@ internal override bool HasSpecialName
             }
         }
 
+        internal override bool HasInternalImplementationOnlyAttribute
+        {
+            get
+            {
+                foreach (var att in this.GetAttributes())
+                {
+                    if (att.IsTargetAttribute("System.Runtime.CompilerServices", "InternalImplementationOnlyAttribute"))
```

Comment:
That would be a strange shape for the language specification. We'll suggest that.